nft_ingester/tests/api_tests.rs

```diff
@@ -505,6 +510,53 @@ mod tests {
         env.teardown().await;
     }
 
+    #[tokio::test]
+    async fn test_api_query_timeout_expected() {
+        let cnt = 20;
+        let cli = Cli::default();
+        let (env, _) = setup::TestEnvironment::create(&cli, cnt, SLOT_UPDATED).await;
+        let pg_pool = env.pg_env.pool.clone();
+
+        let query_timeout = Duration::from_secs(2);
+
+        let result = timeout(query_timeout, query("SELECT pg_sleep(3)").execute(&pg_pool)).await;
+
+        match result {
+            Ok(Ok(res)) => assert!(
+                false,
+                "Query should have timed out, but completed successfully: {:?}",
+                res
+            ),
+            Ok(Err(e)) => assert!(false, "Query should have timed out, but failed: {:?}", e),
```

Comment:
```suggestion
            Ok(Ok(res)) => panic!(
                "Query should have timed out, but completed successfully: {:?}",
                res
            ),
            Ok(Err(e)) => panic!("Query should have timed out, but failed: {:?}", e),
```

Reply:
I'm not sure panic is a good chose for tests. assert! is specifically designed for assertions in tests. If the assertion condition is not met, it will panic! with an indication of what exactly went wrong. This makes tests more readable and understandable.